src/platform_impl/web/script/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
Could we use toplevel gitignore for that?

Reply:
Yes.
The idea was to really not pollute the rest of the codebase with the JavaScript ecosystem.

Would you still prefer it in top-level?

Reply:
it's more discoverable at the top one and you can also explain there why/what ignored and why, so yes.

Reply:
Sure!

Reply:
Done.